src/FsCodec.NewtonsoftJson/OptionConverter.fs

```diff
@@ -8,17 +8,18 @@ open System
 type OptionConverter() =
     inherit JsonConverter()
 
-    override __.CanConvert(t) = t.IsGenericType && t.GetGenericTypeDefinition() = typedefof<option<_>>
+    override __.CanConvert(t : Type) = t.IsGenericType && t.GetGenericTypeDefinition() = typedefof<option<_>>
 
-    override __.WriteJson(writer, value, serializer) =
+    override __.WriteJson(writer : JsonWriter, value : obj, jsonSerializer : JsonSerializer) =
```

Comment:
think `serializer` was a better name, aligning with `writer` etc. elsewhere

Reply:
Everything else was named `jsonSerializer`, that's why I took it :) but I agree, `serializer` is better, will change